### DISCUSSION TITLE
Recommendation for metagenomic assembly when only single-end data are available

### DISCUSSION THREAD

**Discussion by charlesfoster** [ablab/spades#1334] [Q&A] [answered]
Hi,

I'm analysing many samples in a bespoke metagenomic pipeline. One stage of the pipeline involves assembly, with SPAdes being one assembler option. In the case of paired-end reads, the choice is simple: `metaspades.py` (or equivalent `spades.py --meta`). However, what might be the recommendation when only single-end reads are available given that these data are not supported by `metaspades`?

I thought that it might make sense to use `spades.py --sc`, because the single-cell mode was designed for instances (of single-cell data) with uneven coverage:

```
This flag is required for MDA amplified (single-cell) data. Assumes highly uneven coverage and presence of amplification artifacts.
```

While not strictly designed for my purposes, I thought this might be a better option than 'vanilla' spades given that (a) the data represent a range of organisms with uneven genome coverage, and (b) most of the time the data will derived from capture-probe sequencing.

Thoughts would be appreciated. Thanks!

**Comment by asl** [comment, marked as answer]
Well, there are few important differences. In particular, metagenomic mode assumes multiple species with more or less even coverage along the particular genome (so, the local coverage is assumed to be "uniform").

Single-cell mode assumes a single species with very non-uniform coverage. Plus is has special things intended for MDA-imposed artifacts, so it could be overly aggressive in cases w/o such artifacts.

But still, it might be the best option available :)
